src/cogent3/core/sequence.py

```diff
@@ -183,11 +184,11 @@ def counts(
 
         """
         try:
-            data = self._seq
+            data = self._seq.value
```

Comment:
change to
```python
data = str(self._seq)
```
Note the following code is relying on the attribute error of accessing `._seq`